test/formatters/ecto/changeset_test.exs

```diff
@@ -0,0 +1,62 @@
+defmodule DeliriumTremex.Formatters.Ecto.ChangesetTest do
+  use ExUnit.Case
+  doctest DeliriumTremex
+
+  describe "nested changeset formatting" do
```

Comment:
I think that tests would be more understandable if you used descriptive objects in tests. For example `post` & `comment` instead of `foo` and `bar`.

Reply:
Fixed